build/test/integration/shared_css_test.js

```diff
@@ -6,9 +6,11 @@ suite('shared CSS', function() {
   suiteSetup(helper.cleanupWorkspace);
   teardown(helper.cleanupWorkspace);
 
-  test('make APP=test-shared-css, checking shared css are imported',
+  test('make APP=test-shared-css-app, checking shared css are imported',
     function(done) {
-      helper.exec('make APP=test-shared-css', function(error, stdout, stderr) {
+      var command = 'make APP=test-shared-css-app ' +
+        'GAIA_APP_SRCDIRS=build/test/fixtures';
```

Comment:
I saw [1] described we should use GAIA_APP_CONFIG.

[1] https://github.com/RickyChien/gaia/blob/cfa2f00fbf6e3861ed6878b3bc74bd5284989c29/Makefile#L418